### PR TITLE
Feature/add new prop in date input

### PR DESCRIPTION
**Pull request by planivic1**
### Description
in this pr, I add a new prop in date input 

https://user-images.githubusercontent.com/87718261/178333443-d61083e4-d412-4202-a2c3-d764d1976aa8.mp4

### CODE REVIEW

**Review by EvertonLMsilva — DISMISSED** [review]
LGTM

**Comment by alanneves** [comment]
Can u put here an example with placeholder?

File: src/components/Inputs/DateInput/DateInput.vue [hunk 1]
```diff
@@ -1,14 +1,18 @@
 <template>
-  <div class="pb-date-input-container">
+  <div
+    class="pb-date-input-container"
+  >
     <input
-      v-model="inputDate"
+      :v-model="state.typeInput !== 'date' ? placeholder : inputDate"
```
Comment (ixpd) [inline src/components/Inputs/DateInput/DateInput.vue:6]: esta validação/placeholder nao poderia ser adicionado no get da computed?

**Review by ixpd — CHANGES_REQUESTED** [review]

**Review by EvertonLMsilva — DISMISSED** [review]
LGTM

**Review by alanneves — APPROVED** [review]
LOOKS GOOD TO ME

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "vue-polar-bear",
-  "version": "0.10.6",
+  "version": "0.10.7",
```
Comment (victorstassun) [inline package.json:3]: Update this version to 0.10.8 if it´s possible, we are already in the 0.10.7

**Review by victorstassun — CHANGES_REQUESTED** [review]

**Review by ixpd — DISMISSED** [review]
LGTM

**Review by alanneves — APPROVED** [review]
LGTM

**Review by victorstassun — APPROVED** [review]
LGTM

**Review by EvertonLMsilva — APPROVED** [review]
LGTM

**Review by ixpd — APPROVED** [review]

**Pull request merged**